modules/core/02-client/keeper/client_test.go

```diff
@@ -68,6 +68,7 @@ func (suite *KeeperTestSuite) TestUpdateClientTendermint() {
 		name     string
 		malleate func() error
 		expPass  bool
+		freeze   bool // only true if update freezes the client to a new frozen height. false if client is already frozen and header is valid past update
```

Comment:
I refactored this test in #140 to use the new testing package since consensus states were not being stored for all client states resulting status to be `Unknown` on valid cases. 

Could you copy over those changes and adjust?

Reply:
I don't have this function: `suite.chainA.ConstructUpdateTMClientHeaderWithTrustedHeight`